code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -148,8 +148,13 @@
 		if("Mime")
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/mime
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/mime
-	H.equip_or_collect(new suit(H), slot_wear_suit)
-	H.equip_or_collect(new helm(H), slot_head)
+
+	if(H.mind.special_role == SPECIAL_ROLE_WIZARD || !H.mind.special_role == SPECIAL_ROLE_WIZARD_APPRENTICE)
```

Comment:
IIRC, I think you need parentheses on the second bit of this i.e. ` !(H.mind.special_role == SPECIAL_ROLE_WIZARD_APPRENTICE))`.

Reply:
*cough* `!=` *cough*

Reply:
Or if you want to be very very very annoying use `<>`